specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -1671,6 +2157,75 @@
         }
       }
     },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/metrics/{metricName}/groups/{groupName}": {
+      "get": {
+        "tags": [ "Metrics" ],
+        "description": "Gets the server related metrics for a given metric and group combination.",
+        "x-ms-examples": {
+          "service_listMetrics": {
+            "$ref": "./examples/Service_MetricSets.json"
+          }
+        },
+        "operationId": "service_listMetrics",

```

Comment:
Similar to all other operationids, please rename this to `ServiceMetrics_List` and `Metrics_List`

Reply:
Done.